CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 - Add imperial support for UDCF import
 - Desktop: combo box with fresh, sea water and EN 13319 in dive planner
 - Desktop: Changed "salinity" to "water type" at dive planner and dive info
+- Desktop: Small UI fixes
+- Desktop: Change strategy when to allow to delete a cylinder
+- Use correct numeric format for all numbers based on selected locale
```

Comment:
Perhaps shorter: "Format numbers according to system locale"?

(This raises an issue: if I understand correctly, currently the numbers are formatted according to the system locale, not to the language set in the preferences. Should there be a preferences entry for numbers as well?)

Also, wasn't the rule to add these entries to the top of CHANGELOG.md?

Reply:
yes, not adding them on top can cause conflicts.

> Should there be a preferences entry for numbers as well?

i see this more in the lines of format numbers based on language chosen in the preferences.
@atdotde @mturkia opinions?

Reply:
I would think that numbering scheme configured within OS should suffice, even if user switches language within Subsurface. I fail to see why someone would want to use different number formatting in Subsurface than elsewhere in his computer. (Unless he hasn't figured out how to configure it in the OS.)

I would rather not tie the number formatting to the language selection. I.e. I run my Linux in LANG=en_US and LC_NUMERIC=fi_FI.

Reply:
@bstoeger @mturkia 
Did you test the current implementation?
My results for Windows 10, latest master are:
- I switch Subsurface language to "use system default" (German for me) I have commas
- I switch Subsurface language manually to "English" I have dots
- I switch Subsurface language manually to "French" I have commas
- I switch Subsurface language manually to "German" I have commas
Means number format now in fact is tied to the selected Subsurface language and I would keep it like this. Or do we have different results for different platforms? 

@neolit123 
Sorry, it was not my plan to do all this wrong but I simply forgot that there are these guidelines and also misinterpreted the comment in the file. Please let me fix it and also propose a different comment line in CHANGELOG.md which may be more clear (like "add new items at the very top of this file"?!). Will do this later because I'm in a hurry now.
BTW: Could you please point me to where you wrote down your proposal for the formatting standard?

Reply:
i wrote a proposal in a thread on the mailing list called "commit message titles and changelog entries". you can have a look at that, but it didn't get approved by everyone.

Reply:
> My results for Windows 10, latest master are:

Same on Linux.

> i wrote a proposal in a thread on the mailing list

Off-topic: I posted a mail to the list (Subject: snprintf cascades) two days ago - but it never found the way back to me. Can anybody confirm?

Reply:
yes, i cannot find this email.